src/esp/bindings_js/index.js

```diff
@@ -34,20 +35,24 @@ function preload(url) {
 }
 
 Module.preRun.push(() => {
-  let config = {};
-  config.scene = defaultScene;
-  buildConfigFromURLParameters(config);
-  window.config = config;
-  const scene = config.scene;
-  Module.scene = preload(scene);
-  const fileNoExtension = scene.substr(0, scene.lastIndexOf("."));
+  if (window.replayViewerEnabled) {
+    ReplayViewer.preloadFiles(preload);
```

Comment:
Going forward, index.js is the entry point for all our "JS apps" including bindings.html and replay_viewer.html. I do think the choice of files to preload should be specific to the app, so here I delegate preloading to the ReplayViewer class.

I'm open to discussion about how to support multiple JS apps.